test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -163,8 +169,13 @@ public void ChangePointDetectionWithSeasonality()
             // Get predictions
             var enumerator = env.Data.CreateEnumerable<Prediction>(output, true).GetEnumerator();
             Prediction row = null;
+#if NETCOREAPP3_0
```

Comment:
Could you please help me understand the root cause of why netcore 3.0 generates different result?

Reply:
I am curious to know as well :)

Reply:
@tannergooding ?

Reply:
@tannergooding / @sharwell Any updates on this? It would be really good to know why netcore3.0 would generate different results. It may help with other investigations?

Reply:
I appears that previously the tests werent running when .net version was less than 3.0. And @sharwell's fix improves that by getting the tests to run on both again but with different baselines. 

@frank-dong-ms: Can we accept this PR and re-investigate the issue of different baselines later?

@sharwell Can you please amend this PR with an // TODO: [TEST_STABILITY] comment? We are using that tag to track issues in code that still need to be addressed beyond outright test failures?